templates/data-sources/organization_rule.md

```diff
@@ -0,0 +1,42 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "buildkite_organization_rule Data Source - terraform-provider-buildkite"
+subcategory: ""
+description: |-
+  Use this data source to retrieve an organization rule by its ID.
+  More information on organization rules can be found in the documentation https://buildkite.com/docs/pipelines/rules/overview.
+---
+
+# buildkite_organization_rule (Data Source)
+
+~> Rules is a feature that is currently in development and enabled on an opt-in basis for early access. To have this enabled for your organization for utilizing this data source, please reach out to Buildkite's [Support Team](https://buildkite.com/support).
```

Comment:
This is to align with what is currently shown on the [docs](https://buildkite.com/docs/pipelines/rules) - and a pre-requisite before folks can use this data source for their organization.